docs/how-to-guides/meson-args.rst

```diff
@@ -17,5 +16,125 @@ Passing arguments to Meson_
    - Mention the order in which Meson arguments are added, and how it affect
      Meson
 
+Advanced meson options can be accessed by specifying extra arguments to the
+individual meson commands:
+
+   - meson dist: https://mesonbuild.com/Commands.html#dist
+   - meson setup: https://mesonbuild.com/Commands.html#setup
+   - meson compile: https://mesonbuild.com/Commands.html#compile
+   - meson install: https://mesonbuild.com/Commands.html#install
+
+These arguments can be added to the project by adding a section
+to the project's ``pyproject.toml``, or as arguments when building
+the package using the :ref:`build config settings<how-to-guides-config-settings>`
+
+.. tab-set::
+
+    .. tab-item:: pyproject.toml
+        :sync: key_pyproject_toml
+
+		.. code-block::
+
+			[tool.meson-python.args]
+			dist    = ['dist-argument_1', 'dist-argument_2', '...']
+			setup   = ['setup-argument_1', 'setup-argument_2', '...']
+			compile = ['compile-argument_1', 'compile-argument_2', '...']
+			install = ['install-argument_1', 'install-argument_2', '...']
```

Comment:
I think this bit should not be in the tab. IMO we should show how to configure it permanently in `pyproject.toml`, and then how to add more options (temporarily) at build time, with the user's preferred tooling, pypa/build and pip, which is what should be in a tab because they do the same thing.